ui/views/Authentication/ForgotPasswordView.tsx

```diff
@@ -0,0 +1,137 @@
+import { ArrowBack } from '@mui/icons-material';
+import { Button, CircularProgress, TextField } from '@mui/material';
+import classnames from 'classnames';
+import { Link as RouterLink } from 'react-router-dom';
+import { Formik, FormikErrors, FormikHelpers } from 'formik';
+import React, { Fragment, useState } from 'react';
+
+import Logo from 'assets';
+import { useSelector } from 'react-redux';
+import useSendForgotPassword from 'shared/authentication/actions/sendForgotPassword';
+import { getReCAPTCHAKey } from 'shared/bootstrap/selectors';
+import verifyEmailAddress from 'util/verifyEmailAddress';
+import CaptchaMaybe from 'views/Captcha/CaptchaMaybe';
+
+interface ForgotPasswordValues {
+  email: string;
+}
+
+const initialValues: ForgotPasswordValues = {
+  email: '',
+};
+
+export default function ForgotPasswordView(): JSX.Element {
+  const requireCaptcha = !!useSelector(getReCAPTCHAKey);
+  const [verification, setVerification] = useState<string | null>(null);
+  const sendForgotPassword = useSendForgotPassword();
+
+  function validateInput(values: ForgotPasswordValues): FormikErrors<ForgotPasswordValues> {
+    let errors: FormikErrors<ForgotPasswordValues> = {};
+
+    if (values.email) {
+      if (!verifyEmailAddress(values.email)) {
+        errors['email'] = 'Please provide a valid email address.';
+      }
+    }
+
+    return errors;
+  }
+
+  function submitForgotPassword(values: ForgotPasswordValues, helpers: FormikHelpers<ForgotPasswordValues>): Promise<void> {
+    helpers.setSubmitting(true);
+
+    return sendForgotPassword(values.email, verification);
+  }
+
+  return (
+    <Fragment>
+      <div className="absolute top-2.5 left-2.5">
+        <Button
+          className="pr-3"
+          component={ RouterLink }
+          to="/login"
+        >
+          <ArrowBack className="mr-1"/>
+          Back To Login
+        </Button>
+      </div>
+      <Formik
+        initialValues={ initialValues }
+        validate={ validateInput }
+        onSubmit={ submitForgotPassword }
+      >
+        { ({
+             values,
+             errors,
+             touched,
+             handleChange,
+             handleBlur,
+             handleSubmit,
+             isSubmitting,
+             submitForm,
+           }) => (
+          <form onSubmit={ handleSubmit } className="h-full overflow-y-auto pb-20">
+            <div className="flex items-center justify-center w-full h-full max-h-full">
+              <div className="w-full p-10 xl:w-3/12 lg:w-5/12 md:w-2/3 sm:w-10/12 max-w-screen-sm sm:p-0">
+                <div className="flex justify-center w-full mb-5">
+                  <img src={ Logo } className="w-1/3"/>
+                </div>
```

Comment:
This is also used in several spots in the authentication views. This could be lifted into its own component.